### PR TITLE
section left hide issue of  tabs.js

### LINKED ISSUE #216: Block loading within a block leaves section hide

### PR DESCRIPTION
**Pull request by ravverma**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #216 

Test URLs:
- Before: https://main--icicidirect--aemsites.hlx.live/sidekick/blocks/tabs
- After: https://tab-section-fix--icicidirect--aemsites.hlx.live/sidekick/blocks/tabs

### CODE REVIEW

**Comment by anagarwa** [comment]
@ravverma can you please attach a screenshot of the UI issue

**Comment by ravverma** [comment]
before
image


after
image

**Comment by ravverma** [comment]
> @ravverma can you please attach a screenshot of the UI issue

attached

File: blocks/tabs/tabs.js [hunk 1]
```diff
@@ -65,8 +65,11 @@ export default async function decorate(block) {
...
   block.prepend(tablist);
-  block.querySelectorAll('.block.media').forEach((mediaBlock) => {
+  const mediaBlocks = Array.from(block.querySelectorAll('.block.media'));
+  // eslint-disable-next-line no-restricted-syntax
+  for (const mediaBlock of mediaBlocks) {
     decorateBlock(mediaBlock);
-    loadBlock(mediaBlock);
-  });
+    // eslint-disable-next-line no-await-in-loop
+    await loadBlock(mediaBlock);
```
Comment (anagarwa) [inline blocks/tabs/tabs.js:73]: @ravverma if `loadBlock` has some API call, we might be stuck at those API response time. 
Is that something we already accounted while applying this await ?

Comment (ravverma) [reply blocks/tabs/tabs.js:73]: @anagarwa 
the block loading(media) using `observe` it doesn't stuck for the response.

**Comment by ravverma** [comment]
@anagarwa 
As discussed it is a general practice to use `loadBlock` method, So we are good here and it is not adding any blocking time.
Please approve

**Review by anagarwa — APPROVED** [review]

**Pull request merged**